araproc/analysis/spark_power_ratio.py

```diff
@@ -0,0 +1,102 @@
+# this looks for and identifies the spark events by calculating power ratio
+# and comparing with a set threshold 
+
+import numpy as np
+
+from araproc.framework import waveform_utilities as wfu
+
+
+
+
+
+def get_rms_power(waveform):
+  
+    """
+    Calculates RMS power of a waveform.
+
+    Parameters
+    ----------
+    waveform: TGraph
+        A TGraph of the waveform.
+
+    Returns
+    -------
+    power: float
+        RMS power of the waveform.
+    """
+
+    _, trace = wfu.tgraph_to_arrays(waveform)
+    power = np.nanmean(trace**2)
+
+    return power
+
+
+def threshold(threshold_ratio = 5.0):
```

Comment:
I think we can drop this function altogether. Thresholds will be set and assessed in post-processing.

Reply:
Sure thing, just kept it there for future use in case we wanted to set different threshold for separate stations and per configurations. But I can remove it or keep it and terminate its dependencies on other functions.

Reply:
Ya I think that will be done elsewhere, probably better to remove it for now so we don't accidentally implement the threshold at this point.